app/models/manageiq/providers/microsoft/infra_manager/vm.rb

```diff
@@ -1,6 +1,8 @@
 class ManageIQ::Providers::Microsoft::InfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
   include_concern 'ManageIQ::Providers::Microsoft::InfraManager::VmOrTemplateShared'
 
+  supports_not :migrate, :reason => _("Migrate operation is not supported.")
```

Comment:
oh, scratch that, you are right.
although I dont really like this. Because now every Vm supports `migrate` by default unless its explicitly disabled.
This should be the other way round, although I dont have a good idea now, how to implement that.